dot/rpc/subscription/websocket.go

```diff
@@ -367,6 +376,19 @@ func (c *WSConn) initRuntimeVersionListener(reqID float64, _ interface{}) (Liste
 	return rvl, nil
 }
 
+func (c *WSConn) unsubscribeRuntimeVersionListener(reqID float64, l Listener, _ interface{}) {
```

Comment:
You need to register this unsubscribe function to a webscoket method here [./dot/rpc/subscription/subscription.go](https://github.com/ChainSafe/gossamer/blob/bb8021bd45b472f511cf805df2fba2d81ad0293b/dot/rpc/subscription/subscription.go#L45)

Reply:
yes, I've added it.